packages/bot-skeleton/src/services/api/ticks_service.js

```diff
@@ -188,31 +190,27 @@ export default class TicksService {
     }
 
     observe() {
-        this.api.events.on('tick', r => {
-            const {
-                tick,
-                tick: { symbol, id },
-            } = r;
-
-            if (this.ticks.has(symbol)) {
-                this.subscriptions = this.subscriptions.setIn(['tick', symbol], id);
-                this.updateTicksAndCallListeners(symbol, updateTicks(this.ticks.get(symbol), parseTick(tick)));
+        this.api.onMessage().subscribe(({ data }) => {
+            if (data.msg_type === 'tick') {
+                const { tick } = data;
+                const { symbol, id } = data.tick;
```

Comment:
You can get rid of line 195 if you're accessing `symbol` and `id` through `data.tick`.

Reply:
tick used directly in the code 
const { symbol, id } = data.tick; updated to const { symbol, id } = tick;